pom.xml

```diff
@@ -9,7 +9,7 @@
 
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <appengine.target.version>1.9.34</appengine.target.version>
+    <appengine.target.version>1.9.36</appengine.target.version>
```

Comment:
I'm wondering if we need this.  It seems like it's conceivable that we could always use the latest for v1 App Engine APIs and tooling.